modules/cli/src/main/java/org/apache/ignite/cli/commands/configuration/node/NodeConfigShowReplSubCommand.java

```diff
@@ -59,42 +56,23 @@ public class NodeConfigShowReplSubCommand extends BaseCommand {
 
     /** {@inheritDoc} */
     @Override
-    public void run() {
+    public Integer call() {
         var input = NodeConfigShowCallInput.builder().selector(selector);
         if (session.isConnectedToNode()) {
             input.nodeUrl(session.getNodeUrl());
         } else if (nodeUrl != null) {
             input.nodeUrl(nodeUrl);
         } else {
             spec.commandLine().getErr().println("You are not connected to node. Run 'connect' command or use '--node-url' option.");
-            return;
+            return 1;
         }
 
-        CallExecutionPipeline.builder(call)
+        return CallExecutionPipeline.builder(call)
                 .inputProvider(input::build)
                 .output(spec.commandLine().getOut())
                 .errOutput(spec.commandLine().getErr())
                 .decorator(new JsonDecorator())
-                .exceptionHandler(new ShowConfigReplExceptionHandler())
                 .build()
                 .runPipeline();
     }
-
-    private static class ShowConfigReplExceptionHandler implements ExceptionHandler<ApiException> {
```

Comment:
Whey did you remove this handler?

Reply:
Running node returns its config regardless of whether the cluster is initialized or not.